infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -85,6 +87,13 @@ export class AtatWebApiStack extends cdk.Stack {
     // Prevent the GET function from being able to write to DynamoDB (it doesn't need to)
     table.grantReadData(getPortfolioDraftsFn);
 
+    // operationId: createPortfolioStep
+    const createPortfolioStepFn = new lambdaNodejs.NodejsFunction(this, "CreatePortfolioSteptFunction", {
+      entry: "applications/portfolioDrafts/portfolio/post.ts",
```

Comment:
Can we rename this file using the operation name?  It'll be easier to find later.  It's what I started doing in the `portfolioDrafts` directory.
`git mv applications/portfolioDrafts/portfolio/post.ts applications/portfolioDrafts/portfolio/createPortfolioStep.ts`
```suggestion
      entry: "applications/portfolioDrafts/portfolio/createPortfolioStep.ts",
```